src/main/com/fulcrologic/fulcro/algorithms/form_state.cljc

```diff
@@ -116,6 +116,9 @@
     (when (and (not has-fields?) queries-for-config?)
       (throw (ex-info (str "Attempt to add form configuration to " (rc/component-name class) ", but it does not declare any fields!")
                {:offending-component class})))
+    (when (and has-fields? (empty? fields))
```

Comment:
what about subform-keys? They couldbe subforms

Reply:
thanks, I'll look into this.

Reply:
haven't looked into this yet, might have some time in the next few days though